### PR TITLE
Support visionOS

### LINKED ISSUE #5: Support visionOS
Haversack should be able to work with minimal changes on visionOS.

Update the Package.swift to support visionOS, and see if the unit tests can be run on visionOS simulator.

### PR DESCRIPTION
**Pull request by macblazer**
Fixes #5.  Also moved the xcprivacy file into the correct place in the Resources folder.

### CODE REVIEW

**Review by paulinaurbass — APPROVED** [review]

File: Sources/Haversack/Haversack.swift [hunk 1]
```diff
@@ -371,8 +371,8 @@ public struct Haversack {
 
     /// Synchronously delete an item from the keychain that was previously retrieved from the keychain.
     ///
-    /// If the item does not include a `reference` previously retrieved from the keychain: on iOS/tvOS/watchOS all items
-    /// matching the item metadata will be deleted, while on macOS only the first matching item will be deleted.
+    /// If the item does not include a `reference` previously retrieved from the keychain: on iOS/tvOS/visionOS/watchOS
+	/// all items matching the item metadata will be deleted, while on macOS only the first matching item will be deleted.
```
Comment (HarryStrandJamf) [inline Sources/Haversack/Haversack.swift:375]: Looks like this is indented too far

File: Sources/Haversack/Haversack.swift [hunk 2]
```diff
@@ -385,8 +385,8 @@ public struct Haversack {
 
     /// Aynchronously delete an item from the keychain that was previously retrieved from the keychain.
     ///
-    /// If the item does not include a `reference` previously retrieved from the keychain: on iOS/tvOS/watchOS all items
-    /// matching the item metadata will be deleted, while on macOS only the first matching item will be deleted.
+    /// If the item does not include a `reference` previously retrieved from the keychain: on iOS/tvOS/visionOS/watchOS
+	/// all items matching the item metadata will be deleted, while on macOS only the first matching item will be deleted.
```
Comment (HarryStrandJamf) [inline Sources/Haversack/Haversack.swift:389]: It appears like this is indented too far

File: Package.swift [hunk 3]
```diff
@@ -10,6 +10,7 @@ let package = Package(
         .macOS(.v10_13),
         .iOS(.v12),
         .tvOS(.v12),
+		.visionOS(.v1),
```
Comment (HarryStrandJamf) [inline Package.swift:13]: Looks like this is indented too far. Maybe it's just a case of spaces not tabs or vice-versa though. Not a big deal, just looks weird on the PR.

File: README.md [hunk 4]
```diff
@@ -130,12 +130,13 @@ Before submitting your pull request, please do the following:
...
-	2. Four command line invocations:
+	1. In Xcode you can switch destinations to each of the following: "My Mac", any iOS Simulator, any tvOS Simulator, any visionOS Simulator, and any watchOS Simulator.  Run the unit tests for each of the destinations.
+	2. Five command line invocations:
 		- `swift test` to verify macOS functionality.
 		- `xcodebuild test -scheme Haversack-Package -destination 'platform=iOS Simulator,name=iPhone 14'` to verify iOS functionality.
 		- `xcodebuild test -scheme Haversack-Package -destination 'platform=tvOS Simulator,name=Apple TV 4K (3rd generation)'` to verify tvOS functionality.
-		- `xcodebuild test -scheme Haversack-Package -destination 'platform=watchOS Simulator,name=Apple Watch Series 8 (41mm)` to verify watchOS functionality.
+		- `xcodebuild test -scheme Haversack-Package -destination 'platform=watchOS Simulator,name=Apple Watch Series 9 (41mm)'` to verify watchOS functionality.
+        - `xcodebuild test -scheme Haversack-Package -destination 'platform=visionOS Simulator,name=Apple Vision Pro'` to verify visionOS functionality.
```
Comment (HarryStrandJamf) [inline README.md:139]: Another spot where the indentation is different.

**Review by HarryStrandJamf — COMMENTED** [review]
Didn't notice any issues other than it looks like the indentation is weird in several spots. I'll approve since I'm sure it doesn't show up that way in code.

**Review by HarryStrandJamf — COMMENTED** [review]
Hmm...not sure why but it's not giving me the option to approve.

**Review by macshome — COMMENTED** [review]
There are a few strange indentation issues, but nothing I see that prevents approval.

**Pull request merged**